kubernetes/chart/zulip/values.yaml

```diff
@@ -177,6 +177,17 @@ postSetup:
     #   #!/bin/bash
     #   echo "This is a script that gets executed in the Zulip container after installation, once migrations are complete"
 
+  # -- You can add any sidecar to Zulip, like:
+  # sidecars:
+  #   - image: busybox
+  #     name: busy
+  #     command:
+  #       - "/bin/sh"
+  #       - "-c"
+  #       - "echo 'message from sidecar' > /inthis.txt"
+sidecars:
+  []
```

Comment:
Is busybox a realistic sidebar? I'd like to have an example that would be natural for someone to want.

Reply:
That is just a custom placeholder just in many projects as just a placeholder, but I can add another example.

Reply:
Now I added a minio client as a  sidecar instead, I think that would make more sense?